source/gra3-uni4/gra3-uni4-secD-resumen.ptx

```diff
@@ -10,10 +10,40 @@
 <title component="profesor"><custom ref="resumen-leccion-titulo-extraProfe"/></title>
 <!-- ********************************************** -->
 
+<p>En esta sección, dividimos números más grandes y resolvimos problemas en los que hicimos divisiones. Usamos bloques en base diez, diagramas y ecuaciones para representar los números que dividimos. Para ayudarnos a dividir, usamos lo que ya sabemos sobre el valor posicional, sobre grupos iguales y sobre la relación que hay entre la multiplicación y la división.</p>
 
+<p>Por ejemplo, estas son algunas formas en las que podemos encontrar el valor de <m>52 \div 4</m>:</p>
 
-<p>En esta sección, [@@@@@@@@@].</p>
-
+<ul>
+  <li>
+    <p>Poner <m>5</m> decenas y <m>2</m> unidades en <m>4</m> grupos iguales.</p>
+    <image source="svg-source/tikz-file-147697">
+      <shortdescription>Bloques en base diez. 4 grupos de 1 decena y 3 unidades.</shortdescription>
+   </image>
+  </li>
+  <li>
+    <p>Pensar en cuántos grupos de <m>4</m> hay en <m>52</m>.</p>
+    <p>
+      <blockquote><m>10</m> grupos de <m>4</m> forman <m>40</m>.</blockquote>
+      <blockquote><m>3</m> grupos de <m>4</m> forman <m>12</m>.</blockquote>
+      <blockquote><m>13</m> grupos de <m>4</m> forman <m>52</m>.</blockquote>
+    </p>
+  </li>
+  <li>
+    <p>Usar los hechos de multiplicación y escribir ecuaciones.</p>
+    <sidebyside widths="40% 40%" margins="0% 20% 0%" valign="top">
+      <p>
+        <blockquote><m>4 \times 10 = 40</m></blockquote>
+        <blockquote><m>4 \times 3 = 12</m></blockquote>
```

Comment:
Me topé con este `<blockquote>` acá que creo que no debe ir. Los  `<blockquote>` son solo para "citas indentadas". Si quieres una lista de ecuaciones, puedes hacer dos opciones:

#### op1. Sin alinear los `=`
Usa `<line>`, que sirve para especificar el contenido de una linea. Quedaría así:
```xml
<sidebyside widths="40% 40%" margins="0% 20% 0%" valign="top">
      <p>
        <line><m>4 \times 10 = 40</m></line>
        <line><m>4 \times 3 = 12</m></line>
       <p>
....
```

#### op1. Alinear los `=`
Se usa `<md>`. Te puedo dar detalles si eso es lo que necesitas.

Reply:
Enrique, podrías darme los detalles de <md>?

Reply:
Mira acá @oterpater : https://pretextbook.org/examples/sample-article/annotated/section-mathematics.html#section-mathematics-4

(para ver el source en la página, haz click in los links de [view source])